.github/ISSUE_TEMPLATE/3-task.yml

```diff
@@ -0,0 +1,36 @@
+name: "Task"
+description: "Work item"
+title: "[Task]: "
```

Comment:
Question: could we add a set of checkboxes here for the user to pick one of possible labels? For example, the option could be "Documentation", "Refactoring", "Tests".

Reply:
AFAIK this is not possible to implement. The labels are hardcoded per template and not user selectable.

I added a dropdown, but it wont change the label, just the issue text.